pkgs/applications/networking/sniffers/sngrep/default.nix

```diff
@@ -0,0 +1,34 @@
+{stdenv, autoconf, automake, fetchFromGitHub, libpcap, ncurses, openssl, pcre}:
+
+stdenv.mkDerivation rec {
+  name = "sngrep-${version}";
```

Comment:
Using pname here would be good

Reply:
Thanks for that! Do you know if `pname` is documented anywhere? As I gather from [here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/stdenv/generic/make-derivation.nix#L288), the packagename becomes a concatenation of the `pname` and the `version attribute, but I can't find anything about it in the nixpkgs manual.

Reply:
I don't know of any documentation but yes, if you do not specify the name attribute it will be set by 
```
name = "${pname}-${version}"
```